### ISSUE TITLE
Add support for Assign Class command (function code 22)

### ISSUE THREAD

**Issue by atmurray** [dnp3/opendnp3#64]
We're looking to implement (at least in the Masterstation) assign class - function code 22.
Happy to take ownership of this feature

**Comment by jadamcrain** [comment]
Great. Let's figure out the qualifier/objects combinations that need to be sent and where we expose this in the API.

**Comment by atmurray** [comment]
Happy to put that together and make some proposals.

**Comment by jadamcrain** [comment]
Alan, the 2.0.x branch is very stable now. Do you want to look at this or the database stuff?

**Comment by atmurray** [comment]
Absolutely, I'll start with the database stuff as I've already got code for
that.
I've got a train trip tomorrow so I'll get cracking.

**Comment by atmurray** [comment]
This looks fairly straightforward, thinking of something like:

``` c++
    request.SetFunction(FunctionCode::ASSIGN_CLASS);
    // write class object header to specify the class to assign points to
    WriteClassHeaders(request, classes);
    // write data object headers to specify the points to assign
    request.GetWriter().WriteRangeHeader<openpal::UInt16>(QualifierCode::UINT16_START_STOP, gvId, start, stop);
    request.GetWriter().WriteHeader(gvId,QualifierCode::ALL_OBJECTS);
```

Just need to work out how to expose the function as discussed in https://groups.google.com/forum/#!topic/automatak-dnp3/cblIlREiY58

**Comment by jadamcrain** [comment]
Happy to help prototype this with one of the class assignment functions.

**Comment by jadamcrain** [comment]
Implemented in 2.0.x-feature
